explorer/client/package.json

```diff
@@ -54,9 +52,11 @@
     },
     "scripts": {
         "start": "react-scripts start",
-        "start:static": "REACT_APP_DATA=static PORT=8080 react-scripts start",
```

Comment:
Originally, we set `PORT=8080` so that we could view the website with live data on http://localhost:3000 and run tests on http://localhost:8080 at the same time. It would be nice to retain this functionality.

Reply:
I can retain this, although it will _eventually_ go away when static mode goes away. But for now I'll keep it on a separate port.

Reply:
Updated to keep the use port 8080.